CorsixTH/Lua/objects/doors/entrance_right_door.lua

```diff
@@ -36,22 +36,22 @@ class "EntranceDoor" (Object)
 ---@type EntranceDoor
 local EntranceDoor = _G["EntranceDoor"]
 
-function EntranceDoor:EntranceDoor(world, object_type, x, y, direction, etc)
+function EntranceDoor:EntranceDoor(hospital, object_type, x, y, direction, etc)
   self.is_master = object_type == object
```

Comment:
Food for thought: Slave & Master should be deprecated because of their historical background.
I know this isn't your fault actually but may be worth considering if you felt like it.

(Also present at circa line 45)

Reply:
I agree that master/slave in historic context was an extremely horrible idea, but I don't believe trying to erase these words from our language is the right answer.

The reason is that in historic context, the bad part is not master/slave as name for a type of relation that may exist. The bad part here is to enforce something (anything, in my view) onto other beings without their consent or opt-out. The latter is what we should fight, in my opinion. In historic context, the former is just one form of the latter. There are many other forms, eg weapon, war, sex.

Should we simply delete half our dictionary, because lots of things can be applied in a bad way? Does that solve anything? Will people in 5 generations remember and know about the bad things we erase now, so they can avoid them?